eng/config/PublishData.json

```diff
@@ -210,7 +210,7 @@
       "channels": [],
       "vsBranch": "rel/d17.1",
       "vsMajorVersion": 17,
-      "insertionCreateDraftPR": true,
+      "insertionCreateDraftPR": false,
```

Comment:
surprised by this change, isn't 17.1 in QB for the rest of the cycle?

Reply:
Yes, I found there is a check called MSRetailBuild check in the insertion now, which would not run if it is a draft
See this https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/373626

But since all the 17.1 insertion requires qb, even it is not a draft PR it won't be merged.

Reply:
Makes sense! We did some manual work in the insertion tool to run the required checks that existed at the time IIRC. Not very maintainable, so I think this change is reasonable.

Reply:
Yeah, the RIT tool can be updated to also run this test. See https://github.com/dotnet/roslyn-tools/blob/main/src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs#L492-L498